### PR TITLE
Tp1

### PR DESCRIPTION
**Pull request by mihaicostin34**
### Pull Request Overview

This pull request adds TP1

### Testing Strategy

This pull request was tested by running the project

### TODO or Help Wanted

This pull request still needs review

### Build

- [x] Ran `npm build`.

### Author

Signed-off-by: Mihai Costin <mihaicostin342@gmail.com>

### CODE REVIEW

File: docs/tp/01.md [hunk 1]
```diff
@@ -2,4 +2,29 @@
...
+Kotlin offre la possibilité de développement sur plusieurs platformes, étant compatible avec le JVM. Au même temps, toute bibliothèque en Java peut être utilisée dans un programme écrit en Kotlin. L’inverse est aussi vrai. À cause de tout cela, on considère qu’apprendre ce langage peut vous aider aussi à l’avenir, pour vos projets futurs. 
+
+## Syntaxe de base
+Pour commencer à apprendre Kotlin, on vous prie de suivre toutes les chapitres du tutoriel suivant: [Tour de Kotlin](https://kotlinlang.org/docs/kotlin-tour-welcome.html).
+
+## Exercices
+Parcourez les chapitres suivantes et résolvez les exercices: [Exercices](https://play.kotlinlang.org/koans/overview)
+
+## Développement
+Pendant ce semestre, on va utiliser l’IDE Intellij. Pour installer Kotlin, vérifiez le lien dans la séction “Sources”.
```
Comment (CAmi307) [inline docs/tp/01.md:19]:
```suggestion
Pendant ce semestre, on va utiliser l’IDE IntelliJ. Pour installer Kotlin, vérifiez le lien dans la séction “Sources”.
```

File: docs/tp/Images/01.1.png
Comment (alexandruradovici) [inline docs/tp/Images/01.1.png:1]: I suggest using small letters for folder name, like `images`.

File: docs/tp/01.md [hunk 2]
```diff
@@ -2,4 +2,29 @@
...
+
+ Pour créer un nouveau projet en Kotlin, choisissez le langage “Kotlin”:![Nouveau projet en Kotlin](Images/01.1.png)
+Ensuite, vous pouvez commencer à écrire de code.
+
+## Sources
+* [La page principale de Kotlin](https://kotlinlang.org/)
+* [La bibliothèque standard de Kotlin](https://kotlinlang.org/api/latest/jvm/stdlib/)
+* [Kotlin Blog](https://blog.jetbrains.com/kotlin/)
+* [Kotlin pour Android](https://kotlinlang.org/docs/android-overview.html)
+* [Installation d'Intellij](https://www.jetbrains.com/help/idea/installation-guide.html)
```
Comment (alexandruradovici) [inline docs/tp/01.md:29]:
```suggestion
* [Installation d'IntelliJ](https://www.jetbrains.com/help/idea/installation-guide.html)
```

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Comment by alexandruradovici** [comment]
Blocked until #9 is merged.

File: docs/tp/Images/01.1.png
Comment (mihaicostin34) [reply docs/tp/Images/01.1.png:1]: Solved
Image name has the format <tp_number>_<relevant_name>.png

File: docs/tp/01.md (hunk 2 above)
Comment (mihaicostin34) [reply docs/tp/01.md]: Solved

File: docs/tp/01.md (hunk 1 above)
Comment (mihaicostin34) [reply docs/tp/01.md]: Solved

File: docs/tp/01.md [hunk 3]
```diff
@@ -2,4 +2,29 @@
...
+
+Kotlin offre la possibilité de développement sur plusieurs platformes, étant compatible avec le JVM. Au même temps, toute bibliothèque en Java peut être utilisée dans un programme écrit en Kotlin. L’inverse est aussi vrai. À cause de tout cela, on considère qu’apprendre ce langage peut vous aider aussi à l’avenir, pour vos projets futurs. 
+
+## Syntaxe de base
+Pour commencer à apprendre Kotlin, on vous prie de suivre toutes les chapitres du tutoriel suivant: [Tour de Kotlin](https://kotlinlang.org/docs/kotlin-tour-welcome.html).
+
+## Exercices
+Parcourez les chapitres suivantes et résolvez les exercices: [Exercices](https://play.kotlinlang.org/koans/overview)
+
+## Développement
```
Comment (alexandruradovici) [inline docs/tp/01.md:18]: I would place this above the exercises.

**Comment by alexandruradovici** [comment]
Please fill out the pull request message.

File: docs/tp/01.md (hunk 3 above)
Comment (mihaicostin34) [reply docs/tp/01.md:15]: Solved

File: docs/tp/01.md [hunk 4]
```diff
@@ -2,4 +2,29 @@
...
+Ensuite, vous pouvez commencer à écrire de code.
+
+## Exercices
+Parcourez les chapitres suivantes et résolvez les exercices: [Exercices](https://play.kotlinlang.org/koans/overview)
+
+## Sources
+* [La page principale de Kotlin](https://kotlinlang.org/)
+* [La bibliothèque standard de Kotlin](https://kotlinlang.org/api/latest/jvm/stdlib/)
+* [Kotlin Blog](https://blog.jetbrains.com/kotlin/)
+* [Kotlin pour Android](https://kotlinlang.org/docs/android-overview.html)
```
Comment (alexandruradovici) [inline docs/tp/01.md:28]: I would put these two as extra documentation, after the links that are required.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/tp/01.md (hunk 4 above)
Comment (mihaicostin34) [reply docs/tp/01.md:33]: Modified, added necessary links and optional links

**Review by alexandruradovici — APPROVED** [review]

**Comment by alexandruradovici** [comment]
Looks good, @mihaicostin34 please add it to the merge queue if you think it is good to go.

**Pull request merged**